jobs/e2e-test.groovy

```diff
@@ -233,110 +182,11 @@ def runCypressCloud(){
           ],
           propagate: false,
           // The pipeline will NOT wait for this job to complete to avoid
-          // blocking the main pipeline (runLambdaTest).
+          // blocking the main pipeline (runCypressCloud).
```

Comment:
So this job now only exists to spawn another job? It seems like we should either move all the code from `e2e-test-cycloud.groovy` into this file (easy) or update all references to the new job (not as easy as it involves buildmaster changes).

Are there other dependencies that expect the the job that runs cypress to be named `e2e-test-cycloud`?